oss-cloud-backend/serverless.yml

```diff
@@ -47,10 +47,10 @@ functions:
           method: get
 
   getContributorPulls:
```

Comment:
Handler name should reflect what it does.

Reply:
Changed name to updatePullRequests